admin/app/components/target-profiles/tubes-selection.js

```diff
@@ -0,0 +1,169 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { A as EmberArray } from '@ember/array';
+
+export default class TubesSelection extends Component {
+  @service notifications;
+
+  @tracked selectedFrameworkIds;
+
+  @tracked isDownloadModalOpened = false;
+  @tracked tubesWithLevelAndSkills;
+  @tracked downloadContent;
+
+  @tracked selectedTubeIds = EmberArray();
+  @tracked totalTubesCount = 0;
+  @tracked areas;
+  @tracked tubeLevels = {};
+
+  constructor(...args) {
+    super(...args);
+    const pixFramework = this.args.frameworks.find((framework) => framework.name === 'Pix');
+    this.selectedFrameworkIds = [pixFramework.id];
+  }
+
+  get frameworkOptions() {
+    return this.args.frameworks.map((framework) => {
+      return { label: framework.name, value: framework.id };
+    });
+  }
+
+  get selectedFrameworks() {
+    return this.args.frameworks.filter((framework) => this.selectedFrameworkIds.includes(framework.id));
+  }
+
+  get hasNoFrameworksSelected() {
+    return this.selectedFrameworkIds.length === 0;
+  }
+
+  @action
+  setSelectedFrameworkIds(frameworkIds) {
+    this.selectedFrameworkIds = frameworkIds;
+  }
+
+  @action
+  async checkTube(tube) {
+    if (this.selectedTubeIds.includes(tube.id)) {
+      return;
+    }
+    this.selectedTubeIds.pushObject(tube.id);
+
+    await this._triggerOnChange();
```

Comment:
**suggestion:** On pourait utiliser les render modifier plutôt que d'appeler à la main, il faudrait écouter `this.selectedTubeIds`, `this.tubeLevels`, et `this.areas`